addons/web/static/tests/views/list_view_tests.js

```diff
@@ -4012,8 +4012,9 @@ QUnit.module("Views", (hooks) => {
             );
             assert.strictEqual(
                 target.querySelectorAll("tfoot td")[1].textContent,
-                "—",
-                "aggregates monetary should never work if different currencies are used"
+                "",
+                "monetary aggregation should only be attempted with an active aggregation function" +
+                " when using different currencies"
```

Comment:
I would suggest to keep this part but add the ` sum="Sum"` property to the "amount field line 3998 instead and add another assertion for your fix